src/mbgl/sprite/sprite_parser.cpp

```diff
@@ -150,6 +159,33 @@ std::optional<style::ImageContent> getContent(const JSValue& value, const char*
     return std::nullopt;
 }
 
+std::optional<style::TextFit> parseTextFit(const std::string_view& value) {
```

Comment:
Tests covering these two functions should be added to `test/sprite/sprite_parser.test.cpp`.

Make sure to include all branches, including the else branch of the log statement.

Reply:
I see, will add a new test case that could cover parsing the `TextFit`

Reply:
Thanks!

Reply:
Have added the test "SpriteParsingTextFit" to cover the TextFit parsing.

Reply:
OK. Do you also want to test the error logging like the other tests?

Reply:
Oh, you're right, the value of the TextFit may be invalid, I will add another test case to test it.

Reply:
Have added two new test cases to test the invalid and null `TextFit` values.